### DISCUSSION TITLE
How can I add a subdomain to an already created WordPress?

### DISCUSSION THREAD

**Discussion by TipoSamuel** [leokhoa/laragon#406] [Q&A] [answered]
Hello Laragon community.

I am making my portfolio in local with Laragon and I would like it to have a subdomain called demo. How can I get this?

**Comment by Pen-y-Fan** [comment, marked as answer]
Maybe this answer will help explain how Laragon can load different projects using pretty URLs:

- [Pretty URL for .html by default? #404](https://github.com/leokhoa/laragon/discussions/404)

**Reply by TipoSamuel** [reply]
Hey Pen-y-Fan, that was helpful, I was able to solve the problem with your documentation, thanks!
